Std/Data/Bool.lean

```diff
@@ -42,9 +42,13 @@ theorem false_ne_true : false ≠ true := Bool.noConfusion
 
 theorem eq_false_or_eq_true : (b : Bool) → b = true ∨ b = false := by decide
 
-theorem eq_false_iff : {b : Bool} → b = false ↔ b ≠ true := by decide
+-- N.B. The replacement lemma is symmetric to the original so it is
+-- not a direct replacement.
+@[deprecated Bool.not_eq_true]
+theorem eq_false_iff {b : Bool} : b = false ↔ b ≠ true := Iff.of_eq (Bool.not_eq_true b).symm
```

Comment:
The direction here is important because the coercion from `Bool` to `Prop` is `_ = true`. So this direction is the useful one to convert a `Bool` expression to an equivalent `Prop`.